assignment.js

```diff
@@ -31,6 +31,13 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
+let baseNumber = 2
+
+for (let i = 0; i <=12 ; i++) {
```

Comment:
@clo-bot  great use of a `for` loop here. If you interested in how to solve this using a `while` loop - check out the [approach used here](https://github.com/jsd20200127/hw-01-js-basics-control-flow-loops-solution/blob/master/assignment.js#L38)